website/docs/d/cas_certificates.html.makrdown

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Please add link in the website/alicloud.erb for this file.

Reply:
done